style_guides/pl_sql.md

```diff
@@ -1,480 +1,35 @@
 # PL/SQL Style guide
 
-This guide was originally stolen from the [Kickstarter SQL Style Guide](https://gist.github.com/fredbenenson/7bb92718e19138c20591)
+This guide uses parts of and expands upon  the [Kickstarter SQL Style Guide](https://gist.github.com/fredbenenson/7bb92718e19138c20591) and the [Trivadis PL/SQL and & SQL Coding Guidelines](https://trivadis.github.io/plsql-and-sql-coding-guidelines/v4.2/).
 
 ## Principles
 
 * We take a disciplined and practical approach to writing code.
-* We regularly check-in code to Bitbucket
+* We regularly check-in code to source control.
 * We believe consistency in style is very important.
-* We demonstrate intent explicitly in code, via clear structure and comments where needed.
+* We demonstrate intent explicitly in code, via a clear and consistent structure, using comments where needed.
 * We are pragmatic and do not change legacy code for change's sake.
 
-## Rules
+It is split into the following sections:
 
-### General stuff
+1. [Variable Naming Conventions](pl_sql_naming_conventions.md)
+2. [Tables](pl_sql_tables.md)
+3. [Packages](pl_sql_packages.md)
+4. [Triggers](pl_sql_triggers.md)
+5. [Other Objects](pl_sql_other_objects.md)
+6. [Scripts](pl_sql_scripts.md)
+7. [Comments and Change History](pl_sql_comments_history.md)
+8. [Repository Directory Structure](pl_sql_repo_structure.md) 
+9. [Formatting PL/SQL and SQL Code](pl_sql_fomatting_code.md)
+10. [General PL/SQL Writing Guidance](pl_sql_general_guidance.md)
```

Comment:
Should this be item 1 or 2 rather than 10? Feels like this is one of the more common items people will be looking for